quinn/src/recv_stream.rs

```diff
@@ -112,7 +112,16 @@ impl RecvStream {
         Poll::Ready(Ok(buf.filled().len()))
     }
 
-    fn poll_read_buf(
+    /// Attempts to read from the stream into buf.
```

Comment:
Docstring nits: drop the period from the first sentence, and use backticks to quote the code parts (`Poll::Ready(Ok(())`, `Poll::Pending`, `cx.waker()`).